Products/ZenModel/ZentinelPortal.py

```diff
@@ -254,7 +254,7 @@ def setupRoles(self, p):
 
     def setupPermissions(self, p):
         # Set up some suggested role to permission mappings.
-        print "Starting setup of default permissions."
+        #print("Starting setup of default permissions.")
```

Comment:
debug code?

Reply:
removed commented code, and did a PEP8 cleanup